suites/quincy/nvmeof/ceph_nvmeof_deployment.yaml

```diff
@@ -95,6 +97,45 @@ tests:
       name: configure Ceph NVMEoF Gateway
       polarion-id:
 
+# Run IOs using multiple-initiators against multi-subsystems based NVMe-OF targets using fio tool
+  - test:
+      abort-on-fail: true
+      config:
+        gw_node: node6
+        rbd_pool: rbd
+        do_not_create_image: true
+        rep-pool-only: true
+        rep_pool_config:
+          pool: rbd
+        install: true                           # Run SPDK with all pre-requisites
```

Comment:
@preethisn this is actually re-configuring everything- SPDK GW  again on same GW node. I guess we have to skip installation for this test

Reply:
Correct. I didnot notice this. Will address this.

Reply:
Changed accordingly